utils/src/main/scala/za/co/absa/hermes/utils/SparkCompatibility.scala

```diff
@@ -21,7 +21,7 @@ import za.co.absa.commons.version.impl.SemVer20Impl.SemanticVersion
 
 object SparkCompatibility {
   val minSparkVersionIncluded: SemanticVersion = semver"2.4.2"
-  val maxSparkVersionExcluded: SemanticVersion = semver"3.2.0"
```

Comment:
https://github.com/AbsaOSS/spark-commons#spark-version-guard

Maybe we use the spark-commons library and remove this "extra" class?

Reply:
Done. Replaced in commit d1bfcb539af038bfb1f9a016dd568801c2a53ddf.